### PR TITLE
Added Team Member, Skill, and Skill Review info to Home Page

### LINKED ISSUE #17: Add a display of team information on the Home Page
Display
```
Team Members: # (links to Team Page)
Unique Skills: # (links to Skills Page)+
Recent Skill Events: List Last 5 TM_Skill and/or Skill Reviews"
```

### PR DESCRIPTION
**Pull request by Hopding**
Fixes #17. Note that this PR relies on [this API PR](https://github.com/maryvilledev/skilldirectory/pull/145) having been merged in order to display the Team Member and Skill names in the Recent Skill Reviews div.

### CODE REVIEW

**Review by thebho — APPROVED** [review]
LGTM

File: src/Home.jsx [hunk 1]
```diff
@@ -1,9 +1,108 @@
...
+  render() {
+    let skillReviews = null;
+    if(this.state.recentSkillReviews != null) {
+      skillReviews = this.state.recentSkillReviews.map(skillReview =>
+        <li key={skillReview.id}>
+            <div>
+              {skillReview.team_member_name 
+               + ' reviewed the ' + 
+               skillReview.skill_name 
+               + ' skill:'}
```
Comment (solkaz) [inline src/Home.jsx]: Nitpick, but why not use [ES6 template string syntax?](https://github.com/airbnb/javascript#es6-template-literals)

Comment (solkaz) [reply src/Home.jsx]: It's also part of the Airbnb style guide, so it would make sense to start writing code according to it.

**Comment by solkaz** [comment]
:shipit:

**Review by solkaz — APPROVED** [review]

**Pull request merged**